guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -25,40 +25,47 @@ ifdef::foreman-el,katello[]
 include::proc_configuring-repositories-el.adoc[]
 
 +
-. Enable Ruby 2.7 module:
+. If the Ruby 2.5 module has already been enabled, a module reset will need to be performed.
```

Comment:
```suggestion
. If the Ruby 2.5 module has already been enabled, you need to perform a module reset.
```

Reply:
I copied what was already there for PostgreSQL 10, but updated now.